src/main.py

```diff
@@ -8,28 +11,34 @@
 from src.types import DUNE_TO_PG
 
 load_dotenv()
-DUNE_API_KEY = os.environ.get("DUNE_API_KEY")
+DUNE_API_KEY = os.environ.get("DUNE_API_KEY", "INVALID_DUNE_API_KEY")
```

Comment:
This is a cheeky hack. We should have some dedicated env parsing logic that returns a typed structure to the program.

Something similar to `load_config() -> Config` but `load_env(): Env`

Reply:
Yes, a dedicated env loader utility would be good, please also handle required environment variables in that context so we don't have to hack it in like that.

Reply:
I'll do something simpler here to avoid the invalid fallback and will do the env loader at the same time as config loader in an immediate follow up PR.